VideoLocker/res/layout/row_myvideo_course_list.xml

```diff
@@ -4,41 +4,39 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content">
 
-    <LinearLayout
+    <RelativeLayout
         android:id="@+id/course_row_layout"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="@dimen/course_card_height"
         android:background="@color/white"
-        android:descendantFocusability="blocksDescendants"
-        android:orientation="vertical"
-        android:paddingBottom="11dp" >
+        android:descendantFocusability="blocksDescendants" >
 
-        <com.android.volley.toolbox.NetworkImageView
+        <ImageView
             android:id="@+id/course_image"
             android:layout_width="match_parent"
-            android:layout_height="120dp"
-            android:adjustViewBounds="true"
-            android:contentDescription="@string/course_image"
```

Comment:
So should this string resource be removed as well?
> On 10-Sep-2015, at 1:39 pm, Mi@N Kh@LiD <notifications@github.com> wrote:
> 
> In VideoLocker/res/layout/row_myvideo_course_list.xml <https://github.com/edx/edx-app-android/pull/354#discussion_r39136771>:
> 
> >              android:id="@+id/course_image"
> >              android:layout_width="match_parent"
> > -            android:layout_height="120dp"
> > -            android:adjustViewBounds="true"
> > -            android:contentDescription="@string/course_image"
> In voice over, it will be read like "Course Image .. edx Demonstration course and so on" which isn't what we want. We only want it to speak the course name, its date and other text details.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/354/files#r39136771>.
>